src/styles/blocks/card.scss

```
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  align-items: center;
+  width: 200px;
+  box-sizing: border-box;
+  margin-right: 48px;
+  margin-bottom: 46px;
+ 
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &:hover {
+    transform: scale(1.1);
+    transition: 300ms;
+  }
+
+   
+  &__button {
+    height: 40px;
+    border-radius: 5px;
+    background: #00acdc;
+    border: #00acdc;
+   
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #fff;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-decoration: none;
+    box-sizing: border-box;
+  }
+   
+  &__button:hover {
+    height: 40px;
+    border-radius: 5px;
+    background: #fff;
+    border: 1px solid #00acdc;
+    color: #00acdc;
+    transition: 300ms;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+  &__price-label {
+    color: #616070;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price {
+    color: #060b35;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__title h2 {
+    color: #060b35;
+    font-size: 12px;
```

Nesting selector redundant
```suggestion
  &__title {
    color: #060b35;
    font-size: 12px;
```